cmd/aries-agentd/startcmd/start_test.go

```diff
@@ -0,0 +1,344 @@
+/*
```

Comment:
Heads up: Some of the tests here I had moved from main_test.go but I didn't originally write.